conf/routes

```diff
@@ -6,15 +6,15 @@ GET         /robots.txt                      controllers.CachedAssets.at(path="/
 GET         /favicon.ico                     controllers.CachedAssets.busted(path="images/favicons/32x32.ico")
 
 # Editions homepage
-GET         /                                controllers.Homepage.index
+GET         /                                controllers.Homepage.index()
```

Comment:
I have added the brackets as per https://github.com/guardian/support-frontend/pull/3122#discussion_r653618017
allthough I can't remember the exact reason I went for adding the brackets here in place of removing them at the point of use.